index.js

```diff
@@ -57,6 +47,14 @@ module.exports = function wdm(compiler, opts) {
       }
     });
   } else {
+    if (typeof options.filename === 'string') {
+      const filename = options.filename
+        .replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&') // eslint-disable-line no-useless-escape
```

Comment:
Can you add tests?

Reply:
@hiroppy probably already exists, it is just refactor without breaking something, i don't know why it is here :smile:, i want to start rewriting this module, so tests will be in near future

Reply:
ok, thanks